plugins/USBPrinting/USBPrinterOutputDevice.py

```diff
@@ -107,9 +98,16 @@ def _onConfirmExitDialogResult(self, result: bool) -> None:
             application = CuraApplication.getInstance()
             application.triggerNextExitCheck()
 
+    @pyqtSlot(str)
+    def updateFirmware(self, file: Union[str, QUrl]) -> None:
```

Comment:
Shadows built-in file (see previous comment)

Reply:
Fixed in a36deea